tester/test.js

```diff
@@ -30,6 +30,10 @@ const tester = async () => {
     };
     await adapter.run();
 
+    if (array.length == 0) {
+        throw new Error("No entities were returned")
+    }
```

Comment:
I've added an extra check here so there aren't false positives when the array is empty